data/food/recipes.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "metadata": {
+      "date": "2019-01-04",
+      "path": "plain-white-bread",
+      "title": "Plain White Bread",
+      "credits": {
+        "name": "Brown Eyed Baker",
+        "link": "https://www.browneyedbaker.com/white-bread-recipe/"
+      }
+    },
+    "data": {
+      "timingInformation": [
+        { "name": "prep", "value": 20 },
```

Comment:
Personally, I'd probably opt for something more descriptive than value. Maybe `minutes`?

Reply:
Yeah, that's much more descriptive. I've made this change. Thanks